### ISSUE TITLE
Regression task의 loss function 개선 및 탐색

### ISSUE THREAD

**Issue by EJueon** [boostcampaitech4nlp2/level1_semantictextsimilarity_nlp-level1-nlp-10#6]
## 목적
Loss Function을 최적화하여 모델 성능 향상 도모 

## 설명

`self.loss_func = torch.nn.L1Loss()`

기존의 baseline 코드 내에서는 loss function을 MAE(L1 Loss)를 사용하였음
하지만 기본적으로 regression task에서는 loss function을 mean squared error를 사용하거나 Pseudo-Huber loss, Cross-Entropy를 사용함. 

현재 baseline 코드에서 사용되는 loss function은 task 최적의 loss가 아닐 수 있어 개선이 필요해 보임
